templates/cloud_welcome_email.html

```diff
@@ -67,14 +67,14 @@
                                         <div style="position: relative;">
                                             <span>
                                                 <img src="{{.Props.WorkSpacePath}}/static/images/c_avatar.png"
-                                                    style="height: 15px; width: 15px;" alt="c_avatar">
+                                                    style="height: 15px; width: 15px; transform: translateY(2px);" alt="c_avatar">
                                             </span>
                                             <span
-                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40;">
+                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 600; font-size: 16px; line-height: 24px; color: #3D3C40; margin-left: 8px;">
                                                 {{.Props.InviteInfo}}
                                             </span>
                                             <p
-                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40; margin-left: 18px">
+                                                style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 14px; line-height: 20px; color: #3D3C40; margin-left: 26px">
```

Comment:
best practice to add semicolon at the end of the inline-style

Reply:
Thanks Pablo. Fixed the issue.